profile.js

```diff
@@ -0,0 +1,65 @@
+function renderProfile(profile) {
+
+  var $profile = document.createElement('div')
+  var $cover = document.createElement('img')
+  $cover.src = profile.cover
```

Comment:
This is throwing an error. `profile.cover` is `undefined`. Your data has a `coverPhoto` property, not `cover`.